packages/web/src/common/store/pages/history/lineups/sagas.ts

```diff
@@ -48,16 +55,16 @@ function* getHistoryTracks() {
   }
 }
 
-const keepTrackIdAndDateListened = (entry) => ({
+const keepTrackIdAndDateListened = (entry: LineupEntry<Track>) => ({
   uid: entry.uid,
-  kind: entry.track_id ? Kind.TRACKS : Kind.COLLECTIONS,
-  id: entry.track_id || entry.playlist_id,
+  kind: entry.kind,
+  id: entry.track_id,
```

Comment:
I don't think that there can be Playlists in the history page, so these should be fine changes

Reply:
maybe worth checking on this one. If you listen to a playlist, does it not show up?

Reply:
looks like just the tracks